twitterpuller/pom.xml

```diff
@@ -22,6 +22,22 @@
             <groupId>org.twitter4j</groupId>
             <artifactId>twitter4j-stream</artifactId>
         </dependency>
+        <dependency>
```

Comment:
You'll want to add these to the dependency management section of your parent pom

Reply:
They are already managed through the dependency Management of the Spring parent pom.

Reply:
All of them? Interesting - but I recommend overriding that with your own versions, specifically Mockito 3, junit 5.

Reply:
Its currently Mockito version 2.23.4 and JUnit version 4.12 so I will upgrade that.